composer.json

```diff
@@ -12,13 +12,14 @@
   ],
   "require": {
     "php": "^7.2",
-    "nette/application": "~2.4.13",
-    "nette/di": "~2.4.15",
-    "nette/forms": "~2.4.9"
+    "nette/application": "~3.0.0",
+    "nette/di": "~3.0.0-stable",
```

Comment:
Stable?

Reply:
It work's same way as conflict with RC version, it is just more reliable.

Reply:
Switch it to normal version and set the conflict, like in the others.